modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/EditObjectDetails.tsx

```diff
@@ -267,9 +267,20 @@ export default function EditObjectDetails({
 						<ClayPanel
 							collapsable
 							defaultExpanded
-							displayTitle={Liferay.Language.get(
-								'external-data-source'
-							)}
+							displayTitle={
+								<div className="lfr__object-web-edit-object-details-external-data-source-panel">
+									<span className="panel-title">
+										{Liferay.Language.get(
+											'external-data-source'
+										)}
+									</span>
+
+									<div className="lfr__object-web-edit-object-details-external-data-source-panel-container-beta">
+										{values.storageType ===
+											'salesforce' && <BetaButton />}
+									</div>
+								</div>
```

Comment:
Hey @Artartur, could you move the conditional up? Thus avoiding an empty div in cases of storageType other than salesforce. i.e:

```
{values.storageType === 'salesforce' && (
	<div className="lfr__object-web-edit-object-details-external-data-source-panel-container-beta">
		<BetaButton />
	</div>
)}
```

Reply:
Done!